src/na/NAGraphAlgorithms.cpp

```diff
@@ -437,11 +469,14 @@ auto NAGraphAlgorithms::computeSequence(const InteractionGraph& g)
             [&](const auto& u, const auto& v) {
               return g.getDegree(u) > g.getDegree(v);
             });
-  const auto& sequence = groupByConnectedComponent(g, sequenceUngrouped);
-  const auto& [coloring, partialOrder] =
+  auto        sequence = groupByConnectedComponent(g, sequenceUngrouped);
+  const auto& colorEdgesResult =
       colorEdges(g, coveredEdges(g, maxIndepSet), sequence);
-  const auto& fixed   = partialOrder.orderTopologically();
-  const auto& resting = computeRestingPositions(sequence, fixed, coloring);
+  std::unordered_map<Edge, Color, qc::PairHash<qc::Qubit, qc::Qubit>> coloring =
+      colorEdgesResult.first;
```

Comment:
```suggestion
  auto coloring = colorEdgesResult.first;
```
Just a suggestion because the type is so verbose